test/ecto/query/planner_test.exs

```diff
@@ -414,6 +414,13 @@ defmodule Ecto.Query.PlannerTest do
     assert %Ecto.Query.SelectExpr{expr: {:&, [], [0]}} = query.select
   end
 
+  test "prepare: prepare CTEs" do
```

Comment:
We need to add a test that guarantees that we can handle parameter interpolation in both query case and in the fragment case.

Reply:
For example, we need to pass a query with parameters in `as: ^from_query_with_parameter` as well as test a fragment like this `fragment("SOME QUERY WITH ?", parameter)`.